charts/bulk-scan-payment-processor/requirements.yaml

```diff
@@ -2,3 +2,7 @@ dependencies:
   - name: java
     version: ~2.5.0
     repository: '@hmctspublic'
+  - name: servicebus
+    version: ~0.1.1
+    repository: '@hmctspublic'
+    condition: bspp.servicebus.enabled
```

Comment:
should be team `bsp` or explicit name specific for this service, e.g. `payment-processor`

Reply:
bulk-scan-payment-processor = bspp ?

Reply:
`bs-payment-processor`? up for quick survey on our dev channel ;)

Reply:
Igas. changed